### ISSUE TITLE
Machines: Network Interfaces tab UI

### ISSUE THREAD

**Issue by bond95** [cockpit-project/cockpit#7630]
At request of @mareklibra, I was doing Network Interfaces tab for virtual machines. Data from interfaces is complicate to organize, because every tag and attributes are optional except `type` of interface. Thus I prepared concept of how it could look.
1. The most frequently and important data will be in own columns.
2. Less frequently and some specific information will be in `Additional` column (but less then three).
![screenshot from 2017-09-08 13-41-56](https://user-images.githubusercontent.com/3332176/30211163-28c11376-94a0-11e7-88d2-32d016db95a0.png)
What do you think? @garrett

**Comment by petervo** [comment]
@bond95 what would this look like with multiple rows. Since it seems like there may be multiple rows in the additional column for one row in the main columns

**Comment by bond95** [comment]
@petervo yes, maximum three, on the picture (that I send with issue) you can see additional column with two rows in it.

**Comment by petervo** [comment]
@bond95 visually I think that might be a little hard to follow. Can you do a mockup that shows what it looks like with multiple rows in the main columns, instead of just the one 'user' row.

**Comment by bond95** [comment]
@petervo 
![screenshot from 2017-09-12 13-24-01](https://user-images.githubusercontent.com/3332176/30323493-b78b3db4-97bd-11e7-9e6e-8657e0e1810c.png)

**Comment by petervo** [comment]
That doesn't look bad. @andreasn thoughts?

**Comment by andreasn** [comment]
Looks good in general, I think!
 * The headers, such as "Type", "Model type" etc. don't need a colon after them. The colon makes them inconsistent with other areas of Cockpit.
 * What is Count counting?
 * The rows are very tall, even though my monitor is wider than tall. I think it would be best to put MTU and State next to each other when the width allows.
 * Additional is right aligned, but the items under it isn't.

**Comment by garrett** [comment]
Are there any unique identifiers — especially labels — that might differentiate the network devices? It would be best to lead with that as the left-most value. Basically, I mean something like `/sys/class/net/*/` (where the asterisk is the label).

Also, there are basically three states, right... up / down / unknown (DHCP being assigned, network misconfiguration, etc.) — that should be treated like a high-level state value and should be at one end column (either at the very beginning or end). In most cases, I've seen Cockpit display the state as the last value, so that's what I would suggest.

You should be able to get the network states reliably. As an example command on systems I have access to, `grep -H . /sys/class/net/*/operstate` shows me if a network device is up, down, or unknown. Of course, it might be different depending on the API... but I think that's pretty reliable information, right?

**Comment by bond95** [comment]
@andreasn @garrett Okay, so I fixed table according to your requirements.

> What is Count counting?
This is number of network interfaces
![screenshot from 2017-09-13 16-40-11](https://user-images.githubusercontent.com/3332176/30383819-de922ae0-98a2-11e7-98cd-655d42f75ffd.png)

![screenshot from 2017-09-13 16-44-08](https://user-images.githubusercontent.com/3332176/30383802-cd3900ac-98a2-11e7-8e85-f27e83aa876d.png)

**Comment by mareklibra** [comment]
Pull request is merged
